portal-frontend/src/app/features/public/application/submission/pofo-details/pofo-details.component.html

```diff
@@ -77,9 +77,8 @@
     </div>
   </div>
 
-  <div class="full-width">
-    <span class="subheading1">Project Duration</span>
-  </div>
+  <div class="grid-1"></div>
+  <div class="subheading2 grid-double">Project Duration</div>
```

Comment:
Same as above for Project Duration, and POFO soil table also needs to be updated